src/docs/02-reference.md

```diff
@@ -13,5 +13,8 @@ A group of open-source UI components and visual styles to create beautiful, cons
 ## [Federal Election Commision Website Styleguide](https://pages.18f.gov/fec-style/)
 Shared styles and user interface components for the betaFEC website.
 
+## [CALC Style Guide](https://calc-dev.apps.cloud.gov/styleguide/)
+Shared styles and user interface components for CALC.
+
 ## [Glossary](https://github.com/18F/glossary)
-A glossary panel component for your site to help readers understand jargon
\ No newline at end of file
+A glossary panel component for your site to help readers understand jargon
```

Comment:
Weird, I didn't change this line at all. I wonder if GitHub auto-added/removed a CR at the end of the file or something.